src/pages/homepage.tsx

```diff
@@ -93,4 +98,4 @@ const HomePage = ({ mdxContent }: WithMdxResult) => {
 
 export const getStaticProps = withMdx({ load: 'mdx', filename: 'homepage', folder: 'static' })();
 
-export default HomePage;
+export default HomePage;
```

Comment:
Please [keep newlines](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3#) at end of file.